SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -415,8 +416,7 @@
        Conditions:
        * project is a test project -> turn off Roslyn analysis
        * project is excluded -> turn off Roslyn analysis
-       * downloaded SonarLint ruleset and analyzer not found -> no change to existing config
-       * downloaded SonarLint ruleset and analyzer found -> add SonarLint config to existing analysis config
```

Comment:
The old logic was much more complicated: we used to look for specific files in well-known locations and turn analysis on/off based on whether the files existed.The new version just gets the settings from the config file and applies them.